packages/bundler/src/es6-rewriter.ts

```diff
@@ -432,4 +465,60 @@ export class Es6Rewriter {
         importSpecifier,
         {type: 'ImportSpecifier', imported: babel.identifier(exportName)});
   }
+
+  private _rewriteImportMetaToBundleMeta(
+      moduleFile: babel.File,
+      relativeUrl: FileRelativeUrl): babel.File {
+    // Generate a stand-in for any local references to import.meta...
+    // const __bundledImportMeta = {...import.meta, url: __bundledImportURL};
+    // TODO(usergenic): Consider migrating this AST production mishmash into the
+    // `ast` tagged template literal available like this:
+    // https://github.com/Polymer/tools/blob/master/packages/build/src/babel-plugin-dynamic-import-amd.ts#L64
+    const bundledImportMetaName = '__bundledImportMeta';
```

Comment:
We should generate this name to be conflict free with identifiers at `import.meta` use sites.

Reply:
Added issue. https://github.com/Polymer/tools/issues/310